Cargo.toml

```diff
@@ -46,7 +46,9 @@ tracing = {version = "0.1", features = ['log']}
 unicase = "1"
 url = "1"
 www-authenticate = "0.4.0"
+olpc-cjson = "0.1.1"
 
 [dev-dependencies]
 rstest = "0.11"
 hmac = "0.11"
+env_logger = "0.9.0"
```

Comment:
This crate already holds a full dependency on `tracing` with `log` feature enabled, please lean on that and/or take a dev-dependency on `tracing-subscriber` instead:

https://docs.rs/tracing/latest/tracing/#for-log-users
https://docs.rs/tracing-subscriber/latest/tracing_subscriber/fmt/struct.Layer.html#method.with_test_writer

Reply:
Thanks, done.